spec/client_spec.rb

```diff
@@ -113,8 +118,12 @@
   end
 
   context 'put' do
-    it 'returns hash on success with object_id' do
-      expect(client.put_posts(object_id: 1)).to be_a(Hash)
+    it 'returns Takeout::Response on success' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.